app/controllers/api/v1/recordings_controller.rb

```diff
@@ -11,6 +13,13 @@ def index
 
         render_json(data: recordings, status: :ok, include: :formats)
       end
+
+      def recordings
+        recording_sync = RecordingSync.new
+        recording_sync.recording_resync(user: current_user)
```

Comment:
Then you would call the service like `RecordingSync.new(user: current_user).call`